routes/cache.js

```diff
@@ -6,7 +6,7 @@ const jerr = require('../error')
 const bodyParser = require('body-parser')
 const validator = require('validator')
 
-router.use(bodyParser.raw({ type: '*/*' }))
+router.use(bodyParser.raw({ type: '*/*', limit: '50mb' }))
```

Comment:
Is this needed now that `/cache/getitems` exist?

Reply:
The body parser limit is needed mainly for PUT `/cache/items` (multi set), because setting large amounts of keys with their corresponding values (which can be complex stringified objects) require a lot of body size. I tested that for the eth2 use case the required size was around 35mb, so I added some extra margin. Note: this value is the limit after inflating the payload (if the client chooses to send it compressed).